sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/DocumentModelAdministrationAsyncClient.java

```diff
@@ -66,18 +65,19 @@
 import static com.azure.ai.formrecognizer.documentanalysis.implementation.util.Transforms.getComposeDocumentModelRequest;
 import static com.azure.ai.formrecognizer.documentanalysis.implementation.util.Transforms.getInnerCopyAuthorization;
 import static com.azure.ai.formrecognizer.documentanalysis.implementation.util.Transforms.toInnerDocTypes;
-import static com.azure.ai.formrecognizer.documentanalysis.implementation.util.Utility.getComposeModelOptions;
 import static com.azure.core.util.FluxUtil.monoError;
 import static com.azure.core.util.FluxUtil.withContext;
 
 /**
- * This class provides an asynchronous client that contains model management operations
- * that apply to Azure Form Recognizer.
- * Operations allowed by the client are creating, building of custom document analysis models, deleting models,
- * listing models, copying a custom-built model to another Form Recognizer account, composing models from
- * component models, getting operation information and getting resource details.
+ * This class provides an asynchronous client to connect with the Form Recognizer Azure Cognitive Service for
+ * building and managing models.
  *
- * <p><strong>Instantiating an asynchronous Document Model Administration Client</strong></p>
```

Comment:
The Getting started section and champion scenarios samples can be copied here from Package info.